package.json

```diff
@@ -57,6 +57,7 @@
     "bootstrap": "lerna bootstrap --no-ci -- --no-package-lock",
     "commit-init": "commitizen init cz-conventional-changelog --save-dev --save-exact --force",
     "commit": "git-cz",
+    "changelog": "npx ts-node ./scripts/create-release-notes.ts",
```

Comment:
I wasn't able to get `npx` to cooperate in the `tasks.json` so I added a script here instead.

Reply:
You shouldn't need npx here.  npx is how you get access to the npm state from the command line. B/c we're running in an npm script it'll know where to find ts-node. 
```   
 "changelog": "ts-node ./scripts/create-release-notes.ts",
```

Reply:
ah, awesome!